src/transformers/modeling_tf_utils.py

```diff
@@ -662,7 +672,15 @@ def _get_word_embeddings(self, embeddings):
             # TFSharedEmbeddings
             return embeddings.weight
         else:
-            raise ValueError("word embedding is not defined.")
+            embeddings.build([])
```

Comment:
(nit) What exactly happens here? A short comment would be awesome :-)

Reply:
A comment has been added to clarify this.